devsetup/Makefile

```diff
@@ -37,3 +38,26 @@ download_tools: ## Runs an ansible playbook to install required tools with the v
 	ANSIBLE_FORCE_COLOR=true ansible-playbook \
 	-v -i hosts \
 	download_tools.yaml
+
+.PHONY: crc_attach_default_interface
+crc_attach_default_interface:
+	sudo virsh attach-interface crc --source default --type network --model virtio --config --persistent
+
+.PHONY: edpm_compute
+edpm_compute:
+	scripts/gen-ansibleee-ssh-key.sh
+	scripts/gen-edpm-compute-node.sh ${EDPM_COMPUTE_SUFFIX}
+
+.PHONY: edpm_compute_cleanup
+edpm_compute_cleanup:
+	-sudo virsh destroy edpm-compute-${EDPM_COMPUTE_SUFFIX}
+	-sudo virsh undefine edpm-compute-${EDPM_COMPUTE_SUFFIX}
+	rm -f ${HOME}/.crc/machines/crc/edpm-compute-${EDPM_COMPUTE_SUFFIX}.qcow2
```

Comment:
All the interfaces works fine. I have few suggestions which we can do in the new pull request.
I think we can improve the edpm_compute_cleanup to
- also cleanup out/edpm/edpm-compute-0-id_rsa.pub to avoid following issues
```
+ sudo virt-copy-out -d edpm-compute-0 /root/.ssh/id_rsa.pub ../out/edpm
+ mv ../out/edpm/id_rsa.pub ../out/edpm/edpm-compute-0-id_rsa.pub
mv: replace '../out/edpm/edpm-compute-0-id_rsa.pub', overriding mode 0644 (rw-r--r--)? 
```
 and also delete ansibleee-ssh-key-secret secret.
```
oc delete -f ../out/edpm/ansibleee-ssh-key-secret.yaml
```
Rest looks good.